### PR TITLE
Fix UG and DG bugs

### LINKED ISSUE #269: Main Card description UG
image

I think it should be after running get command rather than during get command.

### LINKED ISSUE #270: Commands bolded or not?
image

In this section, commands are bolded. 

image

In other sections, commands are not. Should standardise.

### LINKED ISSUE #287: Outdated examples
image

### LINKED ISSUE #288: Inconsistent usage of '', " " and `  `
Example use '', " " and ` ` interchangeably  for examples

### PR DESCRIPTION
**Pull request by MarcusTXK**
### Changes:
* Update Logic component explanation structure
* Update contributions
* Fix UG bugs

### Issues:
* Resolves #269 
* Resolves #270 
* Resolves #287 
* Resolves #288

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ For any inquiries, please reach us at the email `seer[at]comp.nus.edu.sg`
 [[portfolio](team/marcustxk.md)]
 
 * Role: Integration, Code Quality
-* Responsibilities: In charge of `list` feature
+* Responsibilities: In charge of `list` feature and schedule feature.
```
Comment (YeluriKetan) [inline docs/AboutUs.md:30]: schedule -> `schedule`

File: docs/team/marcustxk.md [hunk 2]
```diff
@@ -21,18 +21,18 @@ Given below are my contributions to the project.
   * Added the ability to list/filter games.
     * What it does: allows the user to list all games stored in gitGud, with the ability to optionally filter by `GAME_ID` containing any of the given keywords.
     * Justification: This feature improves the product significantly because a user can easily view all games they have and apply any filters to easily find specific games.
-  * Added the schedule friend command.
-    * What it does: allows the user save the schedule of their friend to record when they are free or busy during the week.
-    * Challenge: This feature required creating of the `Schedule` and `Day` fields, so the logic (command, parser), model (schedule, day) and storage(Jackson serialization of new fields) packages all had to be updated to support the feature, as well as relevant tests for all the mentioned layers.
+  * Added the schedule friend's availability command.
+    * What it does: allows the user store the schedule of their friend, to record when they are free to play or busy during the week.
```
Comment (YeluriKetan) [inline docs/team/marcustxk.md:25]: user store -> user to store

**Review by tau-bar — APPROVED** [review]
LGTM

File: docs/team/marcustxk.md [hunk 3]
```diff
@@ -44,15 +44,15 @@ Given below are my contributions to the project.
   * Developer Guide:
     * Added target user profiles for gitGud.
     * Added user story `list` feature.
-    * Updated the logic component section of the Developer Guide to follow gitGud.
+    * Updated the logic component section of the Developer Guide to follow gitGud's implementation.
     * Added implementation section of `friend --schedule` feature.
-    * Added missing/Updated existing use cases for all implemented features to be consistent.
-
+    * Added use cases for all existing features Updated existing use cases for all implemented features to be consistent
```
Comment (YeluriKetan) [inline docs/team/marcustxk.md:49]: This sentence doesn't really make sense.

**Review by YeluriKetan — COMMENTED** [review]
Some minor nitpicks.

**Pull request merged**